templates/channels/channel_configuration.haml

```diff
@@ -262,6 +262,35 @@
 
     %hr
 
+  -elif object.channel_type == 'TW'
+    %h4
+      -blocktrans
+        To finish configuring your TwiML REST API channel you'll need to add the following URL in your TwiML REST API instance.
+
+    %hr
+
+    %h4
+      -trans "TwiML REST API Host"
+
+    %p
+      -blocktrans
+        The below endpoint should receive message this channel.
+
+    %code
+      {{ object.config_json.send_url }}
+
+    %h4
+      -trans "Request URL"
+
+    %p
+      -blocktrans
+        Outgoing messages for this channel will be sent to this endpoint.
```

Comment:
This is the wrong way around I think.

Outgoing messages for this channel will be sent to this endpoint. This should be the TwiML REST API host, instead. I.e Rapid Pro will send outgoing messages/calls to this endpoint.